<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,0" columnstretch="0,0" rowminimumheight="0,0,0" columnminimumwidth="0,0">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="RunScanButton">
         <property name="text">
          <string>Run Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_fishing_button">
         <property name="text">
          <string>Start Fishing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_fishing_button">
         <property name="text">
          <string>Stop Fishing</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Screen Shot Lines</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Screen_Shot_Lines_Image">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Screen Shot Image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Screen_Shot_Image">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="threshold1Slider">
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="maximum">
      <number>250</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>threshold 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="threshold2Slider">
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>250</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>threshold 2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>threshold1Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>threshold2Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>threshold2Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>threshold1Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
